psql2mysql/__init__.py

```diff
@@ -436,6 +443,9 @@ def main():
                         check_target_schema(db['target'])
                     cfg.CONF.command.func(cfg, db['source'], db['target'])
 
+        except Psql2MysqlRuntimeError as e:
+            print(e, file=sys.stderr)
+            sys.exit(1)
```

Comment:
Would it make sense to distinguish exit codes from the other ones (e.g. from failure during migration)?

Reply:
Not sure. Would it help us much?